rcon.d.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference types="node" />
```

Comment:
Wasn't these reference comments removed from TypeScript?

Reply:
As far as I know they still work.  Might be deprecated.
It depends on your module structure.